### ISSUE TITLE
Consultation on multi-label sigmoid cross-entropy loss

### ISSUE THREAD

**Issue by wmonica** [PranjalSahu/DRML#2]
Thank you for your code. 
From your code, you just have 2 classes, and the loss function is "tf.nn.softmax_cross_entropy_with_logits". But as mentioned in the paper "Deep Region and Multi-label Learning for Facial Action Unit Detection", the loss function is multi-label sigmoid cross-entropy loss. Could you tell me how to write this part code in tensorflow?

**Comment by PranjalSahu** [comment]
Hello wmonica. Sorry for the late reply. 
For multi label classification you can use 
tf.nn.sigmoid_cross_entropy_with_logits in Tensorflow.
